conanfile.py

```diff
@@ -54,3 +63,6 @@ def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
         if self.settings.os == "Linux":
             self.cpp_info.system_libs.extend(["pthread"])
+
+        if self.options.enable_easy_profiler:
+            self.cpp_info.defines.append('MEDIAGRAPH_USE_EASY_PROFILER')
```

Comment:
it won't be enough, because we use `CONAN_PKG::easy_profiler` in our projects.
This is the downside of using conan targets, in this case code duplication.

Reply:
ho I see. OK.